explorer/templatetags/vite.py

```diff
@@ -1,61 +1,40 @@
-import json
 import os
-from functools import lru_cache
 
 from django import template
 from django.conf import settings
 from django.utils.safestring import mark_safe
 from explorer import app_settings
 
-
 register = template.Library()
 
-VITE_OUTPUT_DIR = "/static/explorer/"
+VITE_OUTPUT_DIR = f"{settings.STATIC_URL}explorer/"
```

Comment:
I've never worked with a project that didn't use `/static` so I didn't even think about that 🤦